examples/starter/webpack.config.prod.js

```diff
@@ -0,0 +1,12 @@
+const path = require('path');
+const createWebpackConfigForProduction = require('@commercetools-frontend/mc-scripts/config/create-webpack-config-for-production');
```

Comment:
also feels really opinionated. Since we are shipping compiled code, there's no reason why they need to use our webpack config right? maybe they want to use parcel 🤷‍♂️

Reply:
Yeah but it is like the easy way to get started. So I think it is fine showing that they can use our config too.

Reply:
It should be possible yes. However, this is meant to be a starter example that you just copy-paste and get started. We can also provide a separate one with parcel, or document it how to use it.
I would take it one step at a time ;)